code/script.js

```diff
@@ -0,0 +1,167 @@
+    <div class="stats"><img class="icon" src=${icons.view}>   ${repo.watchers}</div>
+    <div class="stats"><img class="icon" src=${icons.update}>   ${lastUpdate}</div>
+    </div>
+    `;
+      })
+      .catch((error) => {
+        console.error("repo commits url", error);
+      });
+  });
+  drawChart(sortedRepos);
+  getPullRequests(sortedRepos);
+};
+
+const getPullRequests = (sortedRepos) => {
+  sortedRepos.forEach((repo) => {
+    fetch(`https://api.github.com/repos/technigo/${repo.name}/pulls?per_page=100`)
+      .then((res) => res.json())
+      .then((pulls) => {
+        const userPRs = pulls.filter((pull) => pull.user.login === repo.owner.login);
+        userPRs.forEach((pull) => {
+          const COMMENTS_URL = pull.review_comments_url;
+          const pullDate = new Date(pull.updated_at).toLocaleDateString("nb-NO", { day: "2-digit", month: "2-digit", year: "2-digit" });
+          document.getElementById(`${repo.name}`).innerHTML += `
+        <div id="pull-${repo.name}">
+        <div class="stats"><img class="icon-bigger" src=${icons.pull}>  ${pullDate}</div>
+        </div>`;
+          getComments(COMMENTS_URL, repo);
+        });
+      })
+      .catch((error) => {
+        console.error("getPullRequests", error);
+      });
+  });
+};
+
+const getComments = (COMMENTS_URL, repo) => {
+  fetch(COMMENTS_URL)
+    .then((res) => res.json())
+    .then((comments) => {
+      renderComments(comments, repo);
+    })
+    .catch((error) => {
+      console.error("getComments", error);
+    });
+};
+
+const renderComments = (comments, repo) => {
+  // separating code review comments from user answers
+  const filteredComments = comments.filter((com) => !com.in_reply_to_id);
+  // withdrawing from execution if no comments were available
+  if (filteredComments.length === 0) {
+    return;
+  }
+  const reposStats = document.getElementById(`${repo.name}`);
+  const commentsButton = document.createElement("button");
+  commentsButton.innerText = "Comments";
+  reposStats.appendChild(commentsButton);
+  commentsButton.className = "comments-button";
+  const commentContainer = document.createElement("div");
+  commentContainer.classList.add("comment-container");
+  reposStats.appendChild(commentContainer);
+
+  filteredComments.forEach((comment) => {
+    commentContainer.innerHTML += `
+    <p class="comments">${comment.body}</p>
+   `;
+  });
+  commentsButton.addEventListener("click", () => classToggle(commentContainer));
+};
+
+// showing comments container when active class applied
+const classToggle = (commentContainer) => {
+  commentContainer.classList.toggle("active");
+};
+
+//"go to top" button
+window.onscroll = () => {
+  scrollToTop();
+};
+
+const scrollToTop = () => {
+  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+    toTopBtn.style.display = "block";
+  } else {
+    toTopBtn.style.display = "none";
```

Comment:
Why do you need to hide the button if it's postioned at the bottom of the page all time?